start.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+set -e
+
+node /internals/scripts/validate-config.js
+
```

Comment:
This would only be executed when the container is actually spinning up. I'm not sure if that is too late in our CI/CD setup.

Reply:
The validation step can be added to pipelines in infrastructure repositories as well, if you would like to do these checks earlier.